public/app/plugins/datasource/loki/datasource.ts

```diff
@@ -344,11 +344,9 @@ export class LokiDatasource
    * @returns An Observable of DataQueryResponse with live query results or an empty response if no suitable queries are found.
    * @todo: The name says "backend" but it's actually running the query through the frontend. We should fix this.
    */
-  private runLiveQueryThroughBackend(request: DataQueryRequest<LokiQuery>): Observable<DataQueryResponse> {
-    // this only works in explore-mode so variables don't need to be handled,
+  runLiveQueryThroughBackend(request: DataQueryRequest<LokiQuery>): Observable<DataQueryResponse> {
```

Comment:
I think this can stay private for now, no?
Also, the comment still says 
`* It is intended for explore-mode and logs-queries, not metric queries.`

Reply:
> I think this can stay private for now, no?
I was using it in tests and it throws error that `private` methods can't be used outside of class. But it gave ma an idea that I can use `datasource.query` with `livestreaming: true`. And I don't need to call `runLiveQueryThroughBackend ` and can keep it private. Fixed in 
 https://github.com/grafana/grafana/pull/83831/commits/89410a5753898cdfe314c4cab322fd845bfeaf6e

Reply:
And comment fixed in https://github.com/grafana/grafana/pull/83831/commits/691722fc491c9fd5cb642d97aaa88f3c63bcf462